test/mocha/10-api.js

```diff
@@ -109,12 +109,37 @@ describe('API', function() {
       result.data.should.be.an('object');
       result.data.should.have.property('id');
       result.data.id.should.be.a('string');
+      // NOTE: this might be a bug
+      result.data.should.not.have.property('email');
       result.data.should.have.property('controller');
       result.data.controller.should.be.a('string');
       result.data.controller.should.contain(presentation.holder);
       result.data.should.have.property('capabilityAgentSeed');
       result.data.capabilityAgentSeed.should.be.a('string');
     });
+    it('should authenticate with an existing account', async function() {
+      const {account} = mockData.accounts.authenticate;
+      await helpers.insertAccount({account});
+      const withController = {
+        ...presentation,
+        holder: account.controller
+      };
+      const result = await api.post(
+        '/authenticate', {presentation: withController});
+      should.exist(result);
+      result.status.should.equal(200);
+      result.data.should.be.an('object');
+      result.data.should.have.property('id');
+      result.data.id.should.be.a('string');
+      // NOTE: this might be a bug
```

Comment:
@mattcollier @dlongley have a question here: when calling in `authenticate` with a valid VC and the holder of that via maps to an account with a `controller` `/authenticate` returns the account object with an `email`. Is this the behavior we want? The `/authenticate` test with out an existing account does not return an email because the account created does not have one: 

https://github.com/digitalbazaar/bedrock-vc-issuer/blob/ade562c8cf5bcbeda2a1caca8cbfbe1538109aca/lib/http.js#L49-L64

I guess what I am asking is: should we be returning the email if there is an account with an email set? Just wanted to checked.

Reply:
I had to go down memory lane to figure out what was going on here.  So, this endpoint for a DID login, you can see the schema being used to validate the requests: https://github.com/digitalbazaar/bedrock-vc-issuer/blob/master/schemas/vc-issuer.js#L91-L109

The idea here is that anyone who can provide a valid DID login presentation can effectively setup an account on the system.  There is no opportunity for the user to connect an email or other PII during the DID login CHAPI flow, however, there is no reason that the user, once logged in, cannot provide additional details about themselves that would be attached to their account in the system.

That is to say, if a returning user has added additional information to their account, I think we would want to return that additional information.  There will be no additional information if the account has just been created however.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/49/commits/1b29ddcdf55b5ad9da6f7b6746ccdb5f9e4b7562

Reply:
Related: #56